starknet_py/compile/compiler.py

```diff
@@ -28,24 +28,32 @@ class Compiler:
     def __init__(
         self,
         contract_source: List[StarknetCompilationSource],
+        is_account_contract: bool = False,
         cairo_path: Optional[List[str]] = None,
     ):
         """
         Initializes compiler.
 
         :param contract_source: a list of source files paths
+        :param is_account_contract: Set this to ``True`` to compile account contracts
         :param cairo_path: a ``list`` of paths used by starknet_compile to resolve dependencies within contracts
         """
         self.contract_source = contract_source
+        self.is_account_contract = is_account_contract
         self.search_paths = cairo_path
 
     def compile_contract(self) -> str:
         """
         Compiles a contract and returns it as string
 
+        :raises PreprocessorError: when is_account_contract parameter is incorrectly set
```

Comment:
Shouldn't raise, since we're providing a default? Or will it throw when it's set to true, and contract has a \_\_default\_\_ method?

Reply:
It will throw in 2 cases:
* when trying to compile contract that has `__execute__` method and `is_account_contract=False`
* in the opposite case

See [this line](https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/compiler/compile.py#L159)